openslides_backend/action/actions/chat_group/create.py

```diff
@@ -34,3 +36,12 @@ def get_weight(self, meeting_id: int) -> int:
         if maximum is None:
             return 1
         return maximum + 1
+
+    def check_name_unique(self, instance: Dict[str, Any]) -> None:
+        name_exists = self.datastore.exists(
+            self.model.collection,
+            FilterOperator("name", "=", instance["name"]),
+            lock_result=False,
```

Comment:
this must be locked, since else two requests can create the same group name simultaniously. Why did you set it to `False`?

Reply:
Removed the lock_result = False.